Sources/JXKit/JXContext.swift

```diff
@@ -299,7 +312,54 @@ extension JXContext {
         try JXValue(newErrorFromMessage: "\(error)", in: self)
     }
 
+    /// Attempts to convey the given value into this JavaScript context.
+    ///
+    /// - Throws: `JXErrors.cannotConvey` if the type cannot be conveyed to JavaScript.
+    /// - Seealso: `JXValue.convey` to convey back from JavaScript.
+    public func convey(_ value: Any?) throws -> JXValue {
```

Comment:
Mean to be symmetric with JXValue.convey. Not sure if using the same name is a good idea.